gettsim/policy_environment.py

```diff
@@ -358,9 +361,55 @@ def _load_parameter_group_from_yaml(date, group, parameters=None):
 
     tax_data["datum"] = date
 
+    # Load rounding parameters if they exist
+    if "rounding" in raw_group_data:
+        tax_data["rounding"] = _load_rounding_parameters(
+            date, raw_group_data["rounding"]
+        )
     return tax_data
 
 
+def _load_rounding_parameters(date, rounding_spec):
+    """Load rounding parameters for a specific date from a dictionary.
+
+    Parameters
+    ----------
+    date : datetime.date
+        The date for which the policy system is set up.
+    rounding_spec : dictionary
+        The dictionary that contains rounding parameters
+        for several functions and several dates
+
+    Returns:
+        dictionary: Rounding parameters for the specified date
+    """
+    out = {}
+    rounding_parameters = ["direction", "base"]
+
+    # Load values of all parameters at the specified date.
+    for function_name in rounding_spec.keys():
```

Comment:
Iterate over `.items()` since we are accessing `rounding_spec[function_name]` a couple of times. 

Maybe:

```suggestion
    for function_name, this_rounding_spec in rounding_spec.items():
```
?